Solutions/GreyNoiseThreatIntelligence/Data Connectors/azuredeploy_Connector_GreyNoiseAPISentinel_AzureFunction.json

```diff
@@ -192,7 +193,8 @@
                           "CLIENT_ID": "[parameters('CLIENT_ID')]",
                           "CLIENT_SECRET": "[parameters('CLIENT_SECRET')]",
                           "GREYNOISE_CLASSIFICATIONS": "[parameters('GREYNOISE_CLASSIFICATIONS')]",
-                          "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/Azure/Azure-Sentinel/raw/db458a54839b084eac0e70bbe6e2a41f34f37e2b/Solutions/GreyNoiseThreatIntelligence/Data%20Connectors/GreyNoiseAPISentinelConn.zip"
+                          "GREYNOISE_LIMIT": "0",
+                          "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/Azure/Azure-Sentinel/raw/eda8de6774b9e82e9201babb5d1bba6c8cc80b04/Solutions/GreyNoiseThreatIntelligence/Data%20Connectors/GreyNoiseAPISentinelConn.zip"
```

Comment:
Hello @punkrokk, Replace the link with "https://aka.ms/sentinel-GreyNoise-functionapp"

Reply:
done